src/Stryker.Core/Stryker.Core/Reporters/Json/Location.cs

```diff
@@ -1,27 +1,34 @@
-using Microsoft.CodeAnalysis;
-
 namespace Stryker.Core.Reporters.Json
 {
+    public struct LocationDimensions
```

Comment:
I realize this is pretty much yet another wrapper of the numbers here - I believe eventually things will streamline here a bit so hopefully this is an intermediary step.

Reply:
Couldn't we solve the same problem with overloading for the correct type to use?